### PR TITLE
Fix full app embedding layout

### PR DESCRIPTION
**Pull request by alxnddr**
We've made so many changes to app layout and missed the embedding app layout issues

### Before
Screenshot 2022-04-23 at 03 52 52

### After
Screenshot 2022-04-23 at 03 52 38

## How to verify
- Run Metabase EE
- Enable embedding and setup [our embedding example app](https://github.com/metabase/sso-examples/tree/master/app-embed-example)
- Go through the application without embedding mode ensuring the layout is correct on all pages
- Go through the example application in embedded mode ensuring the layout is correct on all pages

### CODE REVIEW

File: frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx [hunk 1]
```diff
@@ -110,6 +110,7 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)`
   ${({ isSticky, isNavbarOpen }) =>
     isSticky &&
     isNavbarOpen &&
+    !IFRAMED &&
```
Comment (alxnddr) [inline frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx:113]: Without this, dashboard filters in embedded mode look incorrect
Screenshot 2022-04-23 at 03 38 07

File: frontend/src/metabase/query_builder/components/view/View.jsx [hunk 2]
```diff
@@ -456,7 +455,7 @@ export default class View extends React.Component {
 
     // if we don't have a card at all or no databases then we are initializing, so keep it simple
     if (!card || !databases) {
-      return <LoadingAndErrorWrapper className={fitClassNames} loading />;
```
Comment (alxnddr) [inline frontend/src/metabase/query_builder/components/view/View.jsx:459]: `fitClassNames` came from `fitViewport` which was removed a while back, so it was empty

**Comment by alxnddr** [comment]
@metabase-bot run visual tests

**Comment by alxnddr** [comment]
@metabase-bot run visual tests

**Review by ranquild — APPROVED** [review]

**Pull request merged**